packages/program-boilerplate/src/trigger.ts

```diff
@@ -217,13 +222,23 @@ function handleProgramVariableSchemaRequest(
       const errorMes = {
         error:
           "An error occurred in a webtask (PROGRAM_TRIGGER_VARIABLES_SCHEMA_REQUEST)",
-        // consider not returning stack trace for security reasons
         message: e.stack,
       };
 
-      // use winston logger instead
-      console.log(errorMes);
+      sqqtLogger("program-boilerplate").error({
+        message: errorMes.error,
+        ["error.message"]: e.message,
+        ["error.stack"]: e.stack,
+      });
+
+      // FIXME: This code wasn't actually returning a 500 before this, was that meant to be the case
+      // or did we just forget?
+      return {
+        json: errorMes,
+        code: 500,
+      };
```

Comment:
Again not sure if the 204 was previously intentional here or if we just forgot the 500